CHANGELOG.md

```diff
@@ -116,7 +116,9 @@ may elicit a deprecation warning. See [#9086] for more details.
 
 </details>
 
-## v2.3.0 - TBD
+## v2.4.0 - TBD
```

Comment:
I think the new convention, which I like, is not to add this until the "bump" stage.

Reply:
Done.